ush/config_defaults.sh

```diff
@@ -156,7 +156,7 @@ QUEUE_WGRIB2=""
 #
 #-----------------------------------------------------------------------
 #
-USE_CRON_TO_RELAUNCH="FALSE"
+USE_CRON_TO_RELAUNCH="TRUE"
```

Comment:
Do we want to default USE_CRON_TO_RELAUNCH to true?

Reply:
This is wrong. I want to set
"SAVE_CYCLE_LOG="TRUE""